README.md

```diff
@@ -71,27 +74,25 @@ packages:
 
 If you aren't sure which option to take, start with both source and build folders and optimize the `appmap.yml` file later. 
 
-
 ## Recording AppMaps
 
 Once `appmap.yml` is configured for your project, you're ready to record AppMaps. 
 
-
-### Recording mocha test cases:
+### Recording test cases:
 
 1. Validate that the tests run prior to recording AppMaps.
-2. Run `appmap-agent-js` with the `mocha` command and its parameters following the `--` delimiter. For example:
+2. Prefix your test command with `npx appmap-agent-js --` For example:
 ```sh
-npx appmap-agent-js -- mocha test/**/*.ts
+npx appmap-agent-js -- mocha 'test/**/*.ts'
```

Comment:
Why is this document suggesting to use `npx` if it told you to `npm install` it into the project before? Surely if it's installed it's better to use `npm run`.

(I know this isn't changing in this PR, but I thought it might be a good opportunity to consider it.)

Reply:
AFAIK `npm run` fetches scripts in `package.json`. `npx appmap-agent-js` will fail to install anything because it is `@appland/appmap-agent-js` and not `appmap-agent-js`. Maybe you want `npx --no-install appmap-agent-js`?

Reply:
Huh, ok, I thought `npx` and `npm exec` were entirely different. Nevermind then :)